earthpy/tests/test_plot.py

```diff
@@ -16,7 +15,7 @@
 
 # Create tuple
 tuups = (1, 2)
```

Comment:
Instead of defining `tuups` as a module-level variable, it may be better to create such a tuple in the test that it is used (`test_arr_parameter` on line 20).

Reply:
sure. i just created a tuple in place for this particular example. didn't seem to warrant an entire fixture given it's just a few characters. see what you think! :)